### PR TITLE
New package: advaith.CurrencyConverter version 1.2.0

### LINKED ISSUE #171129: [Package Request]: PowerToys-Run-Currency-Converter
### How can we help?

None

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://github.com/Advaith3600/PowerToys-Run-Currency-Converter/releases/tag/v1.2.0
Publisher: advaith3600
Package Name: PowerToys-Run-Currency-Converter
Description: PowerToys Run plugin which will convert real and crypto currencies.
Package Version: 1.2.0
Installer URLs: 
- https://github.com/Advaith3600/PowerToys-Run-Currency-Converter/releases/download/v1.2.0/CurrencyConverter-1.2.0-x64.exe
- https://github.com/Advaith3600/PowerToys-Run-Currency-Converter/releases/download/v1.2.0/CurrencyConverter-1.2.0-ARM64.exe

### PR DESCRIPTION
**Pull request by Advaith3600**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #171129 

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/171128)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @Advaith3600,

This package's files look like a PowerToys plugin. Is `Microsoft.PowerToys` a dependency?

**Comment by Advaith3600** [comment]
Yes, it is a PowerToys Run plugin

File: manifests/a/advaith/CurrencyConverter/1.2.0/advaith.CurrencyConverter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: advaith.CurrencyConverter
+PackageVersion: 1.2.0
+InstallerType: nullsoft
+Installers:
```
Comment (stephengillie) [inline manifests/a/advaith/CurrencyConverter/1.2.0/advaith.CurrencyConverter.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.PowerToys
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
> Yes, it is a PowerToys Run plugin

**Comment by Advaith3600** [comment]
I believe there is something wrong, could anyone help with addressing the issue?

**Comment by stephengillie** [comment]
I had to visit the [package page](https://github.com/Advaith3600/PowerToys-Run-Currency-Converter) to find how to use this. I didn't see it listed in the settings, nor did the shortcuts work.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> {
    "$id": "1",
    "innerException": null,
    "message": "Artifact InstallationVerificationLogs was not found for build 613210.",
    "typeName": "Microsoft.TeamFoundation.Build.WebApi.ArtifactNotFoundException, Microsoft.TeamFoundation.Build2.WebApi",
    "typeKey": "ArtifactNotFoundException",
    "errorCode": 0,
    "eventId": 3000
}

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Package Cache\{c1be0fe8-b34c-4f60-a9a1-939bf696151c}\PowerToysUserSetup-0.84.0-x64.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\Package Cache\F7119E3AB3D5CB3ADCD9A57F0EC227F783695819\MicrosoftEdgeWebview2Setup.exe returned exit code: -1
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\1.3.155.77\MicrosoftEdgeUpdate.exe returned exit code: -1606220282
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\1.3.155.77\MicrosoftEdgeUpdateCore.exe returned exit code: -2147024894
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\MicrosoftEdgeUpdate.exe returned exit code: -1606220282
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\Download\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}\128.0.2739.67\MicrosoftEdge_X64_128.0.2739.67.exe returned exit code: 3
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\128.0.2739.67\BHO\ie_to_edge_stub.exe returned exit code: 1
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\128.0.2739.67\cookie_exporter.exe returned exit code: -1610612722
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\128.0.2739.67\elevation_service.exe returned exit code: 1063
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\128.0.2739.67\msedgewebview2.exe returned exit code: 13
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\128.0.2739.67\msedge_pwa_launcher.exe returned exit code: -2147483645
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\128.0.2739.67\pwahelper.exe returned exit code: 1
> Executable C:\Program Files (x86)\Microsoft\EdgeCore\128.0.2739.67\Installer\setup.exe returned exit code: 3
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\128.0.2739.67\cookie_exporter.exe returned exit code: -1610612722
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\128.0.2739.67\elevation_service.exe returned exit code: 1063
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\128.0.2739.67\msedge.exe returned exit code: 13
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\128.0.2739.67\msedgewebview2.exe returned exit code: 13
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\128.0.2739.67\msedge_pwa_launcher.exe returned exit code: -2147483645
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\128.0.2739.67\pwahelper.exe returned exit code: 1
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\128.0.2739.67\Installer\setup.exe returned exit code: 501
> Executable C:\Program Files (x86)\Microsoft\EdgeWebView\Application\128.0.2739.67\BHO\ie_to_edge_stub.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.ActionRunner.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.Awake.exe returned exit code: -532462766
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.ColorPickerUI.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.CropAndLock.exe returned exit code: 1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.FancyZonesEditor.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.GcodePreviewHandler.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.GcodeThumbnailProvider.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.ImageResizer.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.MarkdownPreviewHandler.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.MonacoPreviewHandler.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.MouseJumpUI.exe returned exit code: -532462766
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.MouseWithoutBorders.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.MouseWithoutBordersHelper.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.PdfPreviewHandler.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.PdfThumbnailProvider.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.PowerAccent.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.QoiPreviewHandler.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.QoiThumbnailProvider.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.ShortcutGuide.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.StlThumbnailProvider.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.SvgPreviewHandler.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.SvgThumbnailProvider.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.Update.exe returned exit code: 1
> Checking if C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesEditor.exe executable file can be started.
> Launching executable file: C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesEditor.exe
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesEditor.exe ran for less than 10 seconds: False
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesEditor.exe returned exit code: -1
> Checking if C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesLauncher.exe executable file can be started.
> Launching executable file: C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesLauncher.exe
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesLauncher.exe ran for less than 10 seconds: True
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesLauncher.exe returned exit code: 1
> Checking if C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesLauncherUI.exe executable file can be started.
> Launching executable file: C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesLauncherUI.exe
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesLauncherUI.exe ran for less than 10 seconds: False
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesLauncherUI.exe returned exit code: -1
> Checking if C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesSnapshotTool.exe executable file can be started.
> Launching executable file: C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesSnapshotTool.exe
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.WorkspacesSnapshotTool.exe ran for less than 10 seconds: True
> Executable C:\Users\validator\AppData\Local\PowerToys\Tools\PowerToys.BugReportTool.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\KeyboardManagerEditor\PowerToys.KeyboardManagerEditor.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\KeyboardManagerEngine\PowerToys.KeyboardManagerEngine.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\Tools\PowerToys.StylesReportTool.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\PowerToys.EnvironmentVariables.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\PowerToys.FileLocksmithUI.exe returned exit code: -1073741189
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\PowerToys.Hosts.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\PowerToys.MeasureToolUI.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\PowerToys.Peek.UI.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\PowerToys.PowerRename.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\PowerToys.RegistryPreview.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\RestartAgent.exe returned exit code: -1073740791
> Executable C:\Program Files (x86)\Microsoft\EdgeUpdate\Download\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}\128.0.2739.67\MicrosoftEdge_X64_128.0.2739.67.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\PowerToys.FileLocksmithUI.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\PowerToys.MouseJumpUI.exe returned exit code: -1
> Executable C:\Users\validator\AppData\Local\PowerToys\WinUI3Apps\RestartAgent.exe returned exit code: -1

Automated error analysis suggests -1073740791 may mean Stack overflow / exhaustion.

Automated error analysis suggests -532462766 may mean EXCEPTION_COMPLUS (Unhandled Exception)

Automated error analysis suggests 1606 may mean `ERROR_UNKNOWN_FEATURE` - Feature ID not registered.

Automated error analysis suggests 1610 may mean `ERROR_BAD_CONFIGURATION` - The configuration data for this product is corrupt.

(Automated response - build 895.)

**Comment by stephengillie** [comment]
> I didn't see [this package] listed in the settings, nor did the shortcuts work.

This is unchanged.

**Comment by stephengillie** [comment]
Hi @Advaith3600,


> The plugin requires Microsoft PowerToys to be installed on the machine. 
Yes, PowerToys is installed. It's inside PowerToys Run settings that I don't see settings for this plugin. 

> Once it's installed the exe file can be ran to install it. 
Yes, this is the order in which the dependency system installs packages.

> The user will have to close PowerToys and open it again to see it working.

Does PowerToys need to be running during this plugin's installation? Because the package manager doesn't currently do this, and might not have the functionality at this time. 

> About the logs you are posting, is it an issue with the exe or the config? Is this something I have to fix? If yes, could you guide me towards some direction?

These logs are the ADO pipeline logs, extracted from the run artifacts, and filtered to include and exclude specific terms. Then strings from the logs are matched against a list of numerical strings, and the corresponding message is included as the automated error analysis.

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Comment by Advaith3600** [comment]
Hello, sorry for the late reply, I was quite busy the past couple of weeks

> Does PowerToys need to be running during this plugin's installation? Because the package manager doesn't currently do this, and might not have the functionality at this time.

The exe can be run while PowerToys is running or not. It won't affect the installation. However, if PowerToys is already running, restarting it is necessary for the plugin to show up in the listing. 

I found [another plugin](https://github.com/microsoft/winget-pkgs/tree/master/manifests/l/lin-ycv/EverythingPowerToys) for PowerToys which has already been added to Winget. Could you please let me know if there is something I have to do from my side?

**Pull request closed without merging**